classes/Meta.php

```diff
@@ -528,9 +519,30 @@ public static function completeMetaTags($metaTags, $defaultValue, Context $conte
             $metaTags['meta_title'] = $defaultValue;
         }
 
+        if (!empty($context->controller) && method_exists($context->controller, 'getTemplateVarPagination')) {
+            $metaTags['meta_title'] = Meta::paginateTitle($metaTags['meta_title']);
+        }
+
         return $metaTags;
     }
 
+    /**
+     * Add page number to title
+     *
+     * @param string $title
+     *
+     * @return string
+     */
+    public static function paginateTitle(string $title): string
+    {
+        $page_num = (int) Tools::getValue('page');
+        if ($page_num > 1) {
+            $title .= ' (' . $page_num . ')';
```

Comment:
```suggestion
            $title .= sprintf(' (%s)', $page_num);
```

It's usually better to format strings with sprintf rather than using concatenation (but I might be nitpicking here)

Reply:
Hello, @matthieu-rolland
Using `sprintf` could be really useful if that was a translatable/editable string. But it is not translatable/editable. So, are you sure sure there are any advantages of using `sprintf` in this particular case?